internal/spokes/spokes.go

```diff
@@ -396,9 +414,44 @@ func (r *SpokesReceivePack) readPack(ctx context.Context, commands []command, ca
 		_ = eg.Wait()
 	}
 
+	if waitErr := cmd.Wait(); waitErr != nil {
+		return waitErr
+	}
+
 	return nil
 }
 
+func (r *SpokesReceivePack) checkFsckConfig() (bool, error) {
+	receiveFsck, err := config.GetConfig(r.repoPath, "receive.fsckObjects")
+	if err != nil {
+		return false, err
+	}
+	transferFsck, err := config.GetConfig(r.repoPath, "transfer.fsckObjects")
```

Comment:
Is it worth introducing a helper for this, e.g. `config.GetEntry` (or maybe a better name), that returns either an empty string or the last entry in the list?  It seems like a common pattern below to need that.

Reply:
I didn't do it because it was only used at two places for now, and I was planning to add it as a part of a future pull request, but since you've already asked here we go :D [2eee4c4](https://github.com/github/spokes-receive-pack/pull/21/commits/2eee4c4e5d772d7140392d4a9583e072d2b8e3c1)